src/static/actions/cards.js

```diff
@@ -173,3 +182,162 @@ export function createCard(token, values) {
             });
     };
 }
+
+export function cardEditFailure(error, message) {
+    return {
+        type: CARD_EDIT_FAILURE,
+        payload: {
+            status: error,
+            statusText: message
+        }
+    };
+}
+
+export function cardEditRequest() {
+    return {
+        type: CARD_EDIT_REQUEST
+    };
+}
+
+export function cardEditSuccess() {
+    return {
+        type: CARD_EDIT_SUCCESS
+    };
+}
+
```

Comment:
May be lighter with arrowed functions

Reply:
I squashed these functions in a generic one in the next branch.